src/main/lrsql/init/git_data.clj

```diff
@@ -0,0 +1,10 @@
+(ns lrsql.init.git-data
+  (:require [clojure.edn :as edn]
+            [clojure.string]
+            [clojure.java.shell :refer [sh]]))
+
+(defn read-version []
+  (try (edn/read-string (slurp "lrsql/config/git-details.edn"))
+       (catch Exception _
+         (try (clojure.string/trim (str "DEV-" (:out (sh "git" "describe" "--tags"))))
```

Comment:
i suspect in this case since you are attempting to have a fallback in the case of a non-exact tag that this is where you would want the hash `(sh "git" "rev-parse" "HEAD")`. If we try here at all, that seems more useful than "last tag"